docs/getting-started/auth-self-defence-checks.adoc

```diff
@@ -0,0 +1,148 @@
+= Self Defence Checks
+
+As part of the Auth SDK module, there is a self defence check feature.  This feature offers multiple checks that can be used to analyse the secuirty of the device.
+
+== Prequisite's
+
+. Import and initialise the Auth SDK module - link:auth.adoc[AeroGear Services Auth SDK].
+
+== Usage
+
+To begin using the self defence checks in your app, you'll need to retrieve an instance of `SecurityService`. The `SecurityService` can be retrieved using the `MobileCore#getInstance` method.
+
+[source, java]
+----
+SecurityService securityService = MobileCore.getInstance(SecurityService.class);
+----
+
+Any subsequent `MobileCore#getInstance` method will return the same instance of `SecurityService`.
+
+=== Invoking a Single Self Defence Check
+The `SecurityService#check` method can be used to run a single secuirty check.  It expects a `SecurityCheckType` paramater and returns `SecurityCheckResult`.
+
+[source, java]
+----
+SecurityCheckResult result = securityService.check(SecurityCheckType.<check_type>);
+----
+
+See all available `SecurityCheckType` link:auth-self-defence-javadoc.adoc[here].
+
+=== Invoking Multiple Self Defence Checks
+
+==== Synchronously
+
+To run multiple self defence checks _synchronously_, you'll first need to invoke the `SecurityService#getCheckExecutor` method.  This methods returns `SyncSecurityCheckExecutor` where checks can be added (chained) to the executor and executed synchronously.
+
+
+[source, java]
+----
+SyncSecurityCheckExecutor syncCheckExecutor = securityService.getCheckExecutor();
```

Comment:
This section could already include examples of using the `addCheck()` method to illustrate the chaining.